bool misiSatu_mulai = True
bool misiDua_mulai = False
bool misiTiga_mulai = False
bool misiEmpat_mulai = False

bool misiSatu_selesai = False
bool misiDua_selesai = False
bool misiTiga_selesai = False